virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc

```diff
@@ -12,8 +12,44 @@ If you import a VMware template, the wizard creates a virtual machine based on t
 include::modules/technology-preview.adoc[leveloffset=+1]
 :!FeatureName:
 
+== Creating and using a VMware Virtual Disk Development Kit image
+
+The VMware Virtual Disk Development Kit (VDDK) SDK is used to copy VMware virtual disks. You can create a VDDK image and add it to the the `v2v-vmware` ConfigMap to import a VMware virtual machine.
+
+You can store the VDDK image in the internal {product-title} image registry or in a secure external image registry.
+
+[NOTE]
+====
+Storing the VDDK image in a public registry may violate the terms of the VMware license.
+====
+
 .Prerequisites
 
+* You must configure an image registry.
+
+=== Accessing the internal image registry
+
+You can access an {product-title} registry directly, from within the cluster, or externally, by exposing the registry with a route.
+
+.Internal access
```

Comment:
Not critical, but in this case I would probably have used a different format, like:

```
Internal access:: 
You can access the internal image registry xref:../../../registry/accessing-the-registry.adoc#registry-accessing-directly_accessing-the-registry[directly, from your cluster].
```

Again, it's fine the way it is, the headings just looked a little odd to me as they were, since a lot of the times those heading types are for Procedure and Prerequisites, etc.

Reply:
LGTM